src/nodes/softmax.h

```diff
@@ -105,7 +139,7 @@ class Softmax : public Node {
 			dst <<               idx <<"++ ) {" << std::endl;
 		}
 		INDT_2 << "output"<< idxs << " = ";
-		dst           << expfunc << "(input" << idxs << "-max);" << std::endl;
+		dst           << expfunc() << "(input" << idxs << "-max);" << std::endl;
```

Comment:
expfunc is a std::string? I'd say the parenthesis is not valid syntax. But tests pass.
What is happening here?

Reply:
Ah, nvm, it's the new function above...